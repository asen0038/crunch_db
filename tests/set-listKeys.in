set a 1 2 3
set a 9 8 5
get a
list keys
list entries
bye
