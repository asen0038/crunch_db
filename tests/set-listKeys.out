>
ok

>
ok

>
[9 8 5]

>
a

>
a [9 8 5]

>
bye
